### DISCUSSION TITLE
Import existing tasks to board

### DISCUSSION THREAD

**Discussion by amurena** [mgmeyers/obsidian-kanban#525] [Ideas & Feature Requests]
I'm not sure if this is possible, but it would be great to populate your kanban board with existing tasks spread across notes in your database.

Use Case: I capture tasks within my notes on a topic, either on my daily note or a dedicated note as well. It would be great for those tasks to be populated onto the kanban board so I can manage the completion of them within the board itself.

**Comment by gmkado** [comment]
There's another plugin that does something like that https://github.com/roovo/obsidian-card-board

**Comment by ingvald** [comment]
tnx for the plugin tip. also, saw a feature request for this: https://github.com/mgmeyers/obsidian-kanban/issues/467
